htdocs/router.php

```diff
@@ -0,0 +1,90 @@
+<?php
+error_log("URL:  " . $_SERVER['REQUEST_URI']);
+if (preg_match('#^/bootstrap/(.*)#', $_SERVER["REQUEST_URI"])){
+	// Bootstrap is not a Loris module, don't rewrite it.
+	// RewriteRule ^bootstrap/(.*) bootstrap/$1 [L]
+	return false;
```

Comment:
Return false makes it so that no action is taken on the request, and executes it as direct path request